src/Domain/Triggers.hpp

```diff
@@ -0,0 +1,98 @@
+// Distributed under the MIT License.
```

Comment:
I think this should be in a less generically named file.

Reply:
Maybe `Evolution/Triggers/SeparationLessThan`?

Reply:
Yes, let's get it out of Domain to avoid a new dependency there.